prediction_market_agent_tooling/tools/betting_strategies/kelly_criterion.py

```diff
@@ -11,6 +11,66 @@
 
 OutcomeIndex = t.Literal[0, 1]
 
+from enum import Enum
+
+
+class BetOutcome(str, Enum):
+    YES = "Yes"
+    NO = "No"
+
+
+def check_is_valid_probability(probability: float) -> None:
+    if probability < 0 or probability > 1:
```

Comment:
```suggestion
    if not 0 <= probability <= 1:
```

little python magic showcase, no need to commit 😄